Cargo.toml

```diff
@@ -8,20 +8,20 @@ log = "0.3"
 log4rs = "0.6"
 docopt = "0.7"
 getopts = "0.2"
-chrono = { version = "0.3", features = ["serde"] }
+chrono = { version = "0.4.5", features = ["serde"] }
 lazy_static = "0.2"
 regex = "0.2"
 url = "1.4"
 rust-crypto = "^0.2"
 threadpool = "1.3"
-iron = "0.5"
-router = "0.5"
-bodyparser = "0.6"
-persistent = "0.3"
-logger = "0.3"
+iron = "0.6"
+router = "0.6"
+bodyparser = "0.8"
+persistent = "0.4"
+logger = "0.4"
 rustc-serialize = "0.3"
-serde = "0.9"
-serde_derive = "0.9"
-serde_json = "0.9"
-consul = "0.0.6"
+serde = "1.0.71"
+serde_derive = "1.0.71"
+serde_json = "1.0.26"
+consul = {git = "https://github.com/stusmall/consul-rust.git"}
```

Comment:
can't we depend on a specific version?

Reply:
Yes.

1. `chrono` need new version 
2. `consul` client is not support new consul server
3.  Keep updating

Reply:
Sorry, factotum binary is my mistake, I will delete it.